src/cts/src/TreeBuilder.cpp

```diff
@@ -73,6 +73,44 @@ void TreeBuilder::initBlockages()
                   "{} placed hard macros will be treated like blockages.",
                   bboxList_.size());
   }
+
+  // add tree buffer width and height for legalization
+  std::string buffer;
+  if (!options_->getTreeBuffer().empty()) {
+    buffer = options_->getTreeBuffer();
+  } else {
+    buffer = options_->getRootBuffer();
+  }
+  odb::dbMaster* libCell = db_->findMaster(buffer.c_str());
+  assert(libCell != nullptr);
```

Comment:
If I remember correctly, assert does nothing when the code is compiled in release mode. Perhaps it's safer to have an if-else clause checking the nullptr, and throwing an error if it's null.

Reply:
You're absolutely correct.   This is a sloppy short-cut.  However, a proper and safe error handling requires a lot of work for flow control.    Can we postpone this to later?   Right now, basic flow control in CTS is completely lacking.  For example, running the same command twice in a row will cause a fatal.

Reply:
Fine by me. Could you add a TODO comment to remind us of the spots where it should be fixed?